setup.cfg

```diff
@@ -3,3 +3,5 @@ max-line-length = 120
 ignore = E203,E501,W503
 per-file-ignores =
   __init__.py:F401
+exclude =
+    venv
```

Comment:
This is only needed because of your specific workflow that uses a virtualenv?

Reply:
It is recommended to use a virtual environment for a project. There could be different configurations for this virtenv. I've added here a typical one. It also works as a placeholder to help with adding other specific configurations.

Reply:
> It is recommended to use a virtual environment for a project.

Not everyone is using specifically virtual environments to isolate installations, e.g. we are using conda environment instead. And the location is pretty specific although I supposed it's the commonly used naming convention.